docs/source/installation.md

```diff
@@ -1,11 +1,17 @@
 # Installation
 
-First, download the latest Streamflow compatible release for your platform from the [releases](https://github.com/livepeer/go-livepeer/releases) page. If you are using OSX, download `livepeer_darwin.tar.gz`. If you are using Linux, download `livepeer_linux.tar.gz`.
+First, download the latest mainnet compatible release for your platform from the [releases](https://github.com/livepeer/go-livepeer/releases) page. If you are using OSX, download `livepeer_darwin-amd64.tar.gz`. If you are using Linux, download `livepeer_linux-amd64.tar.gz`.
+
+To download the file using wget:
+
+```
+$ wget https://github.com/livepeer/go-livepeer/releases/download/v0.5.3/livepeer-<YOUR PLATFORM>.tar.gz
```

Comment:
Let's change `v0.5.3` to <RELEASE_VERSION> since the latest release version will be constantly changing.